exp/urls.py

```diff
@@ -29,6 +28,7 @@
     ParticipantListView,
     PreviewProxyView,
     RenameVideoView,
+    ResearcherDashboardView,
```

Comment:
this we *could* leave as ExperimenterDashboardView but it's more ambiguous whether it's the app or the type of user - fine either way

Reply:
I am going to leave it, because the works already done.  Let me know if you change your mind.